docker/dpcpp/Dockerfile

```
@@ -0,0 +1,22 @@
+# DPC++ version (git revision) to install
+ARG IMPL_VERSION
+
+# TODO Change namespace
+FROM psalz/sycl-cts-ci-common:latest
+
+RUN git clone https://github.com/intel/llvm.git \
+      --branch=sycl --single-branch --shallow-since=2021-09-01 \
+      --recurse-submodules /tmp/dpcpp && \
+    cd /tmp/dpcpp && \
+    git checkout $IMPL_VERSION
+
+RUN python3 /tmp/dpcpp/buildbot/configure.py \
+      --src-dir=/tmp/dpcpp \
+      --obj-dir=/tmp/build \
+      --build-type=Release \
+      --cmake-opt=-DCMAKE_INSTALL_PREFIX=/sycl && \
+    python3 /tmp/dpcpp/buildbot/compile.py \
+      --src-dir=/tmp/dpcpp \
+      --obj-dir=/tmp/build
```

```suggestion
      --obj-dir=/tmp/build \
      -j$NCPU
```
I think it make sense to set # of parallel executors.

---

Ah, actually that environment variable no longer exists. I used to set it based on `getconf _NPROCESSORS_ONLN`, however I since then switched to building with Ninja, which automatically determines the number of available cores. I will remove it for hipSYCL as well.